Sources/Subs-Db-mysql.php

```diff
@@ -100,7 +100,7 @@ function smf_db_initiate($db_server, $db_name, $db_user, $db_passwd, $db_prefix,
 	if (empty($db_options['dont_select_db']) && !@mysqli_select_db($connection, $db_name) && empty($db_options['non_fatal']))
 		display_db_error();
 
-	mysqli_query($connection, 'SET SESSION sql_mode = \'ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION\'');
+	mysqli_query($connection, 'SET SESSION sql_mode = \'ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION,NO_BACKSLASH_ESCAPES\'');
```

Comment:
For the sake of easier reading, could you change this to use double quotes around  the overall string? Then it could look like this:
`mysqli_query($connection, "SET SESSION sql_mode = 'ONLY_FULL_GROUP_BY,STRICT...`

Reply:
Both are good idea,
will change it later on.

Reply:
To get a better understanding i split the modes

Reply:
So why aren't we using `$smcFunc` for this?

Reply:
First, Why we should?
Secondly, it doesn't work.

Reply:
- it handles escaping things for us
-    no? It accepts a connection object as the third param

Reply:
ther is no escaping, you would use the raw method --> provide nothing
feel free to do a pr with this changes.

The result would be, that the mysql connection get unstabler and slower --> got no problem with this.